cpg_workflows/stages/gatk_sv/gatk_sv_single_sample.py

```diff
@@ -116,6 +125,31 @@ def queue_jobs(self, sequencing_group: SequencingGroup, inputs: StageInput) -> S
 
         expected_d = self.expected_outputs(sequencing_group)
 
+        if only_jobs := config_retrieve(['workflow', self.name, 'only_jobs']):
+            # if only_jobs is set, only run the specified jobs
+            # this is useful for samples which need to re-run specific jobs
+            # e.g. if manta failed and needs to be re-run with more memory
+
+            # disable the evidence collection jobs if they're not in only_jobs
+            if 'coverage_counts' not in only_jobs:
+                input_dict['collect_coverage'] = False
+            if 'pesr' not in only_jobs:
+                input_dict['collect_pesr'] = False
+
+            # disable the caller jobs that are not in only_jobs by nulling their docker image
+            for key, val in input_dict.items():
+                if key in [f'{caller}_docker' for caller in SV_CALLERS]:
+                    caller = key.removesuffix('_docker')
+                    input_dict[key] = val if caller in only_jobs else None
+
+            # remove the expected outputs for the jobs that are not in only_jobs
+            new_expected = {}
+            for job in only_jobs:
+                for key, path in expected_d.items():
+                    if job in key:
+                        new_expected[key] = path
+            expected_d = new_expected
```

Comment:
You can move this logic up into expected_outputs() for consistency - i.e. only return the outputs dictionary with the outputs we intend to generate in this run. That will mean that the `expected_outputs()` is always limited to the things we actually intend to run, which will be used in the DAG generation.

e.g. 
- only_jobs = ['manta']
- manta VCF already exists
- expected_outputs -> includes pesr, coverage and manta
- workflow.py spots that pesr.vcf doesn't exist, so it runs queue_jobs
- in queue_jobs we tell it to only run manta, and launch the WDL
- WDL runs, manta is regenerated, overwrites the manta file we already had